docs/migration-guides/index.mdx

```diff
@@ -17,15 +17,17 @@ These migration guides help you more effectively use Qiskit and Qiskit Runtime:
    * [Overview of Qiskit v1.0 packaging changes](./metapackage-migration)
    * [Qiskit v1.0 feature changes](./qiskit-1.0-features)
 * [Migrate to the Qiskit Runtime V2 primitives](./v2-primitives)
-* [Qiskit Runtime execution mode changes](./sessions)
-* Migrate to Qiskit Runtime
+* [Migrate to local simulators](./local-simulators)
+* [Execution mode changes](./sessions)
+* Migrate from `backend.run` to primitives
    * [Overview](./qiskit-runtime)
    * [Migrate from `qiskit-ibmq-provider`](./qiskit-runtime-from-ibmq-provider)
    * [Migrate from `qiskit_ibm_provider`](./qiskit-runtime-from-ibm-provider)
+   * [Migrate using backend primitives](./qiskit-backend-primitives)
```

Comment:
I think I'm seeing that the titles aren't consistent for this file. As a general rule, the following must be the same: the title in the metadata, the title in the index.mdx tile, and the title in the _toc.json file. (We sometimes break this rule when titles are inconveniently long for the toc!)